src/internal/operations/inject/inject.go

```diff
@@ -37,6 +42,12 @@ type (
 			ctr *count.Bus,
 		) (*details.Details, error)
 
+		IsServiceEnabled(
```

Comment:
Since this is declared twice, it should get moved into its own interface, then you can embed that one within these other two.